cardano-cli/test/Test/CLI/Shelley/Golden/TextEnvelope/Certificates/OperationalCertificate.hs

```diff
@@ -4,71 +4,60 @@ module Test.CLI.Shelley.Golden.TextEnvelope.Certificates.OperationalCertificate
   ( golden_shelleyOperationalCertificate
   ) where
 
-import           Cardano.Prelude
-
 import           Cardano.Api.Typed (AsType (..), HasTextEnvelope (..))
-
+import           Cardano.Prelude
 import           Hedgehog (Property)
-import qualified Hedgehog as H
 
-import           Test.OptParse as OP
+import qualified Test.OptParse as OP
```

Comment:
IMO the qualified import of `Test.OptParse` takes away from the readability of these tests. If we were in a situation where we imported several modules or there were name clashes I'd say fair enough. I don't think this is the case here though. @intricate @erikd thoughts?

Reply:
Good coding styles usually say to use an import list or use qualified imports so that it's clear where the function comes from.

I feel that excessive use of import lists clashes with the strict flags we use that ban unused imports.

With import lists, the experience for me has been that I've had to add and remove functions to and from this lists often repeatedly.  For example if I wanted to add some debugging, I import the function I need, and then when I'm done, I need to remove them again.

A different story happens with refactoring, that doesn't concern the use of our strict flags.

When I copy files from another file to use as a template, I need to fiddle with import lists, whereas with qualified imports it's less so.  Qualified imports results in more refactor friendly code for some kinds of refactoring.

Reply:
But, I agree it does look a little more readable without the qualified imports 😂

Reply:
All very fair points!